### ISSUE TITLE
Mixed Content Error

### ISSUE THREAD

**Issue by MikeNGarrett** [dasevilla/rdio-party#110]
I'm getting a playback error in the latest version of Chrome. It doesn't appear to relate to any action taken inside the app. 

I initially thought this was a mixed content issue (http vs https), but it appears this is happening on all tracks. 

I didn't see anything fishy in the console, either. Here's the track that was having issues (I think). 
```
{"playState":1,"playingTrack":{"id":"-Jp43-HuePFbPWCb9Ueu","timestamp":"2015-05-11T22:19:47.336Z","trackKey":"t50644953","userKey":"s2559","votes":{"s2559":"like","s699613":"like"}},"position":177}
```

I'll keep an eye out for more instances.

**Comment by MikeNGarrett** [comment]
I got reports from a couple of users that they experienced playback errors that required a refresh to correct. The song starts playing, the album art comes in and then everything goes away. 

Also, they said the following, "I think there are sync issues from time to time, if you're behind and the party track ends, it just skips to the next."

**Comment by rstecker** [comment]
I spotted an NPE for me at one point.  This is the code snippet where the error happened:

          var data = _.extend({
            'track': response.result[self.rdioTrackKey],
            'formattedDuration': self.getDuration(response.result[self.rdioTrackKey].duration),  <---- this line broke
            'masterUser': userName,
            'addedBy': addedByName,
            'favorited': favorited
          });

This might also be tied to the region issue.  But we need to scrub the player code to handle bad data better and that should prevent the whole section from blanking out.

**Comment by MikeNGarrett** [comment]
I don't think it has to do with the region since it will play after refresh. When I was working with the js API on another project I was having similar issues. It seems to me that it has to do with connection issues between Rdio and the end user through the API. That's just a feeling though. It will take some digging to find out if that's correct, but that's the first place I would look.

**Comment by FreeForm-Radio** [comment]
I'm not sure if this will be of any help, but when I'm running W7 with latest Firefox, that artwork disappears and I have to click refresh (as stated). I run this on an old XP machine at work. The sound stops but the artwork and buttons stay. I click on the musical note button and the song resumes.

**Comment by FreeForm-Radio** [comment]
Here's what I've found from testing the party over the weekend. 

1. When I host the party, I have to click the resume playback or take master control button after each song. Listeners do not have to do that. I'm not having to refresh to make the songs start. 

2. When we "commandeered" another party (afternoon vibes), the songs mostly played without interruption. There were some that would start and after about 1 second, the Now Playing panel would disappear and refreshing was necessary. One song (Peace Train by Cat Stevens) did not play after restarting. We gave that song enough time to complete and the next song did start and play properly. (I did check and peace train played on the regular rdio site.) 

3. At 5:55 am pacific time, a song started, played about a minute and then I had to the take master control button to start the song back. 

Not sure if you have logfiles or not, but the activity would have occurred between 5am and 5:40 am pacific time on 5-25-15. 

Let me know if there any tests I can run that might be helpful to isolate the bug.
